distributed/scheduler.py

```diff
@@ -1817,6 +1872,7 @@ def __init__(
             self._tasks = tasks
         else:
             self._tasks = dict()
+        self._computations = deque(maxlen=100)
```

Comment:
I believe, this should be configurable from the start. We've seen in the past that too long deques can cause problems for long running clusters and reducing the deque length is often the only way to deal with this problem.

We do not know where people are issuing computations from or how large those modules are. With large modules, these `Computation` objects might easily reach MB size (for reference, our scheduler.py has ~260KiB) since we might be storing multiple code snapshots per `Computation`

Reply:
No disagreement from me